frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -101,6 +101,16 @@ class ManageAccounts extends Component<Props, State> {
               this.toggleAccount(account.code, !active)
                 .then(() => event.target.disabled = false);
             }} />
+          Watchonly:
```

Comment:
Should this be localized?

Reply:
this is just a placeholder for the frontend guys to clean up - i don't expect the final version to contain this word.